generators/generator-constants.js

```diff
@@ -27,29 +27,29 @@ const DOCKER_MONGODB = 'mongo:4.0.12';
 const DOCKER_COUCHBASE = 'couchbase:6.0.0';
 const DOCKER_CASSANDRA = 'cassandra:3.11.4';
 const DOCKER_MSSQL = 'microsoft/mssql-server-linux:latest';
-const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.4';
-const DOCKER_MEMCACHED = 'memcached:1.5.16-alpine';
+const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.5';
+const DOCKER_MEMCACHED = 'memcached:1.5.17-alpine';
 const DOCKER_KEYCLOAK = 'jboss/keycloak:6.0.1'; // The version should match the attribute 'keycloakVersion' from /docker-compose/templates/realm-config/jhipster-realm.json.ejs and /server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs
 const DOCKER_ELASTICSEARCH = 'docker.elastic.co/elasticsearch/elasticsearch:6.4.3'; // The version should be coerent with the one from spring-data-elasticsearch project
-const DOCKER_KAFKA = 'confluentinc/cp-kafka:5.3.0';
-const DOCKER_ZOOKEEPER = 'confluentinc/cp-zookeeper:5.3.0';
+const DOCKER_KAFKA = 'confluentinc/cp-kafka:5.3.1';
+const DOCKER_ZOOKEEPER = 'confluentinc/cp-zookeeper:5.3.1';
 const DOCKER_SONAR = 'sonarqube:7.9.1-community';
 const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v4.1.0';
 const DOCKER_JHIPSTER_CURATOR = 'jhipster/jhipster-curator:v4.1.0';
 const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v4.1.0';
 const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v4.1.0';
 const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'jhipster/jhipster-import-dashboards:v4.1.0';
 const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.1.0';
-const DOCKER_TRAEFIK = 'traefik:1.7.14';
-const DOCKER_CONSUL = 'consul:1.6.0';
+const DOCKER_TRAEFIK = 'traefik:1.7.15';
+const DOCKER_CONSUL = 'consul:1.6.1';
```

Comment:
there is a problem with consul:

```
➜ docker pull consul:1.6.1
1.6.1: Pulling from library/consul
no matching manifest for linux/amd64 in the manifest list entries
```

Reply:
Yes I saw it now, but it is available in dockerhub !?!

Reply:
I will roolback it now...

Reply:
available at Docker Hub but it seems broken: https://hub.docker.com/_/consul

Reply:
They created it again...
I will include it again to see if now it is OK

Reply:
@pascalgrimaud This seems OK now!